### DISCUSSION TITLE
[fr] delete comments edit history

### DISCUSSION THREAD

**Discussion by throwaway242685** [community/community#143764] [Mobile]
hi, could you please implement an option to delete our comments edit history in the Android app?

![Screenshot_2024-11-06_11-25-50_1.png](https://github.com/user-attachments/assets/affc7789-448c-4274-861d-b17e99a2de7d)

this option is already implemented in the GitHub website, but it's missing from the Android app.

**Comment by bronglil** [comment]
To request the addition of the option to delete comment edit history on the GitHub Android app, you can reach out to GitHub Support or submit feedback directly through the app. Here’s a quick guide on how to do it:

In-App Feedback:

Open the GitHub app on your Android device.
Tap on your profile picture or Settings menu.
Look for an option like Send Feedback or Help & Support (the exact wording may vary).
Describe the request clearly: mention that the feature to delete comment edit history, which is available on the GitHub website, is missing on the Android app. Attach the relevant screenshot if possible to help illustrate your request.
GitHub Support:

Go to the [GitHub Support Contact Page](https://support.github.com/contact) on a browser.
Select Feature Request or Mobile App Issue (if available).
Provide a brief description, noting that this feature is available on the web version but missing on the Android app, and that you’d like it added for consistency.
GitHub Community Discussion:

You can also check or post on the [GitHub Community Forum](https://github.community/) to see if others have requested this feature and to show support for its addition.
GitHub regularly updates its mobile apps, so submitting your feedback can help get this feature prioritized in future updates.

**Reply by throwaway242685** [reply]
but... those things redirect me here, to this community page.
